### ISSUE TITLE
[Bug]: ramps doesn't work 

### ISSUE THREAD

**Issue by MeisterAH** [HGLabor/issues#55]
### Welcher Client wurde verwendet?

NoRiskClient

### Beschreibung

map: 
<img width="875" alt="image" src="https://github.com/user-attachments/assets/a228b225-4ff0-4ee9-b043-14cd46216e93">

ist aber glaube ich bei allen rampen so

**Comment by RadFlunder** [comment]
Was meinsten mit Ramps doesn't work genau? Also das man einfach in die Rampe laufen kann und sie keine Hitbox hat oder irgend was anderes?

**Comment by RadFlunder** [comment]
@MeisterAH

**Comment by MeisterAH** [comment]
Ja, das war irgendwie ganz komisch an dem Tag. Da hat irgendwie nichts funktioniert. (Die Issues https://github.com/HGLabor/issues/issues/54 und https://github.com/HGLabor/issues/issues/53 waren auch an dem Tag) Ich weiß nicht, ob NoRisk nur etwas getestet hat oder ob es ein seltener Bug war, aber ich hatte es danach auch nie wieder so deshalb ja.

**Comment by MeisterAH** [comment]
> Was meinsten mit Ramps doesn't work genau? Also das man einfach in die Rampe laufen kann und sie keine Hitbox hat oder irgend was anderes?

wenn man auf die raps gelaufen ist ist man halt gestorben (man kam halt nicht hoch)

**Comment by RadFlunder** [comment]
Dann denk ich mal, dass das so ist wie du gesagt hast... Die barriers und barrier_slabs werden ja im Code gesetzt und dann sollte es eigentlich unmöglich sein, dass die im train spawnen. Der hat ja gar nicht ne function dafür.
